package cf.rodolfo.JavaCore.O_Exception.execption.domain;

public class Employee extends Person {
	public void save() {
		System.out.println("Saving employee");
	}
}
